cartridge/shop/models.py

```diff
@@ -129,6 +141,8 @@ def save(self, *args, **kwargs):
         if updating and not settings.SHOP_USE_VARIATIONS:
             default = self.variations.get(default=True)
             self.copy_price_fields_to(default)
+        elif hasattr(self, 'content_model'):
+            self.content_model = self._meta.object_name.lower()
```

Comment:
Why should the default content_model be `None`? A few ideas come to mind but maybe you have more.

- Users can have their own subclass named "Product".
- If we ever need to implement logic in `ContentTyped` that distinguishes between default and non-default content models, we'll have an easy way to check.